pkg/windowsdesktop/pkg/Directory.Build.props

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
     <FrameworkListFileClass Include="Accessibility.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="Microsoft.VisualBasic.dll" Profile="WindowsForms" />
```

Comment:
Do we need to add `Microsoft.VisualBasic.Forms` here?

Reply:
I tried doing that I'm getting errors. I'm going to add that in a separate step to see what happens next.